release.env

```diff
@@ -0,0 +1,3 @@
+RELEASE_DIR="<Release dir on server>"
```

Comment:
I think this could be included in the script itself. Only the token has to be in a non-committed file. We'd only run this in one place anyways. OTOH, not that important, and I see I've done the same with config in `domjudge@domjudge.org:cov-submit-data-*.sh`.

Reply:
its now in the script.